app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -103,7 +103,7 @@ def check_enrollment(enrollment)
     enrollment_outcomes[:enrollments_checked] += 1
 
     response = proofer.request_proofing_results(
-      enrollment.unique_id, enrollment.enrollment_code
+      enrollment,
     )
```

Comment:
I think we will want to update `app/services/usps_in_person_proofing/mock/proofer.rb` to only accept one param as well.

Reply:
Done! 👍🏻 TY.